r/R/marker_heatmap.R

```diff
@@ -9,9 +9,14 @@
 runMarkerHeatmap <- function(req, data) {
   nFeatures <- req$body$nGenes
   cellSets <- req$body$cellSets$children
+  cell_ids <- req$body$cellIds
 
-  top_markers <- getTopMarkerGenes(nFeatures, data, cellSets)
+  cell_sets_ids <- lapply(cellSets, function(x) x[["cellIds"]])
+
+  top_markers <- memoisedGetTopMarkerGenes(nFeatures, data, cellSets, cell_sets_ids)
```

Comment:
same, singular `set` when `cell_set_ids`

```suggestion
  top_markers <- memoisedGetTopMarkerGenes(nFeatures, data, cellSets, cell_set_ids)
```

Reply:
discussed in https://github.com/biomage-org/worker/pull/73#discussion_r1129813278